src/Analysis/Ast/Impl/Analyzer/Evaluation/ExpressionEval.Generics.cs

```diff
@@ -25,6 +25,7 @@
 using Microsoft.Python.Core;
 using Microsoft.Python.Parsing;
 using Microsoft.Python.Parsing.Ast;
+using Microsoft.Python.Parsing.Definition;
```

Comment:
We don't use 'Definitions' if interface is declared in `Definitions` subfolder. Just same namespace as the rest. Grouping in `definitions` folder is just a convenience thing.

Reply:
OK